cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSISink.java

```diff
@@ -443,7 +447,7 @@ protected Accumulator getRollbackedAccumulationForRetry() {
      * Rollbacks the accumulation once more.
      * @param rollbackedAccumulation
      */
-    protected void doRollbackAgain(Accumulator rollbackedAccumulation) {
+    protected void doRollbackAgain(Accumulator rollbackedAccumulation) {        
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1385/commits/07a7b2349d27c6801078b2717cdd83c0a9bac56e